entity.go

```diff
@@ -77,6 +77,26 @@ func NewMultipart(header Header, parts []*Entity) (*Entity, error) {
 	return New(header, r)
 }
 
+// ReadOptions represents the options in a message reader
+type ReadOptions struct {
+	Header textproto.ReadOptions
```

Comment:
What about just making this field anonymous?

```go
type ReadOptions struct {
	textproto.ReadOptions
}
```

Reply:
yes, thanks for the suggestion.